Identity.BusinessLogic/IAuthenticationService.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Identity.Core.Models;
+
+namespace Identity.BusinessLogic
+{
+    public interface IAuthenticationService
+    {
+        Task<string?> AuthenticateAsync(Login loginCredentials);
+        Task<bool> RegisterUserAsync(Register registerCredentials);
```

Comment:
The models should be on the DAL layer, and on the BLL layer there should be DTO, which you will transfer to the services